clients/cli/src/workers/online.rs

```diff
@@ -153,74 +206,111 @@ async fn attempt_task_fetch(
     environment: &Environment,
     client_id: &str,
 ) -> Result<(), bool> {
+    // Record fetch attempt
+    state.record_fetch_attempt();
+
     let _ = event_sender
         .send(Event::task_fetcher_with_level(
-            "[Task step 1 of 3] Fetching tasks... Note: CLI tasks are harder to solve, so they receive 10 times more points than web provers".to_string(),
+            "[Task step 1 of 3] Fetching task... Note: CLI tasks are harder to solve, so they receive 10 times more points than web provers".to_string(),
```

Comment:
The note here will become obsolete with selective task difficulty, right? Should we maybe change it from `they receive 10 times more points` to `they receive more points`?